gammapy/background/background_estimate.py

```diff
@@ -0,0 +1,56 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from .ring import ring_area_factor
+from ..region import find_reflected_regions
+
+__all__ = [
+    'BackgroundEstimate',
+    'ring_background_estimate',
+    'reflected_regions_background_estimate',
+]
+
+
+class BackgroundEstimate(object):
+    """Cointainer class for background estimate
+
+    This is meant to hold the result from a region based background estimation
+    for one observation.
+
+    Parameters:
```

Comment:
Remove the colon here. Otherwise this will give a warning and make the Sphinx build fail on travis-ci.